src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -15,7 +15,8 @@
     SupplierAddressId INT NULL,
     SupplierContactId INT NULL,
     CatalogueSolutionsViewed BIT DEFAULT 0,

```

Comment:
I've added tech debt task 8531 to remodel this, but it's OK for now

Reply:
👍